_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,85 @@
+
+## 2. Create an overview for events and fairs
+
+Throughout the year, ePages is organizing internal events like our [hackathon](/blog/events/this-years-hackathon-a-travel-through-time/){:target="_blank"}, our Sales Kick-off event, or the [ePages YOU](/blog/events/epages-you-2019-from-a-perspective-of-two-newbies/){:target="_blank"}.
+But ePages is also participating in external events such as conferences and fairs.
+These events can also provide content for your company Instagram channel.
+In order to keep them in in mind, create an overview of all internal and external events.
+You should also note down which team or employee visits the external events.
+Thus, you can remind them in time to take some nice pictures of the event.
+
+## 3. Create a content plan
+
+In order to coordinate your content on Instagram, you can create a content plan.
+This content plan can be understood as a calendar in which you can note down which picture you'd like to share on which day.
+That makes it easier for you to plan your content sharing in advance and agree on it within your team.
+
+But it didn't work for us.
+We prefer to be flexible and react to the current situation.
+So, we're deciding each week what content we would like to share (removed Instagram here to no repeat it again).
+Nevertheless, we keep the content plan in mind - it might support us in the future.
+
+## 4. Consistent visual presentation of your Instagram profile
+
+Create a profile photo which fits to the corporate design of your company.
+This will establish a stronger connection between your profile and your company.
+If you have a highlights section, create cover images that that match the company's corporate design as well.
+These points will support a consistent visual presentation of your Instagram profile.
+And of course, also keep in mind to have a consistent visual presentation of the Instagram feed.
+
+## 5. Be up-to-date
+
+If your Instagram channel is supposed to represent your company culture, the images should show what you're up to as a company and how the daily working live of your employees looks like.
+So last, but not least, you should always be up-to-date about the company live.
+Ask colleagues of different departments if there's anything new you can share on the Instagram channel and keep an overview of what is going on.
+
+These are some actions we take to manage our Instagram channel.
+Do have any other ideas on how to keep a company Instagram channel running? Let us know. 😊
```

Comment:
Would differ a bit more here: Do you have other ideas and concepts to run your company Instagram channel?

What do you think about either linking our Instagram or our Twitter account behind "Let us know". That way we'd directly tell them how they should approach us.